src/components/pages/profile/surveyAnswers/surveyAnswersBox.js

```diff
@@ -16,11 +13,11 @@ const SurveyAnswersBox = ({index}) => {
  
   
    return (
-    <Paper className={classes.paper}>
+    <div style={{position: 'absolute', bottom: 0, marginBottom: 20, marginTop: 20}}>
```

Comment:
Because of the position absolute on the small screens, this div comes on top of the Profile info. Try to change that so it's not overlaping